docs/source/machconf.rst

```diff
@@ -195,6 +196,14 @@ All fields are mandatory, unless indicated otherwise below.
 * ``pre_bootstrap_0``: list of commands to execute for initialization of main agent (optional).
 * ``pre_bootstrap_1``: list of commands to execute for initialization of sub-agent (optional).
 * ``forward_tunnel_endpoint``: name of the host which can be used to create ssh tunnels from the compute nodes to the outside world (optional).
+* ``virtenv_mode``: RADICAL-Pilot agent enviroment setup (optional):
+        * ``create``: create a python virtual enviroment from scratch (default).
+        * ``recreate``: delete the exsiting virtual enviroment and build it from scratch, if not found then ``create``.
+        * ``use``: use an existing virtual enviroment, if not found then ``create``.
+        * ``update`` update the existing virtual enviroment, if not found then ``create``.
+        * ``local``: use the client existing virtual enviroment.
+* ``virtenv``: only use when ``virtenv_mode=use``, path to an exsiting python enviroment with pre-installed RCT stack (optional). 
```

Comment:
Is `virtenv_dist` still a thing?

Reply:
Alas it will remain a thing for a while...

Reply:
@andre-merzky ping

Reply:
`By default, RP will install virtualenv version '1.9' while bootstrapping the pilot.  setting 'virtenv_dist` to a different version string will use that respective version.  Setting it to the string 'system' will use the system supplied 'virtualenv' command.`